src/app/route/MyAccountDetails/MyAccountDetails.component.js

```diff
@@ -0,0 +1,630 @@
+                          type="password"
+                          label="Confirm New Password"
+                          id="confirmPassword"
+                          validation={ ['notEmpty', 'password'] }
+                        />
+                        <button block="MyAccountDetails" elem="Submit" type="submit">Save New Password</button>
+                    </fieldset>
+                </Form>
+            </>
+        );
+    }
+
+    /**
+     * Render Account Information block
+     */
+    renderAccountInformation() {
+        const { customer } = this.props;
+        const {
+            firstname,
+            lastname,
+            email,
+            is_subscribed
+        } = customer;
+
+        return (
+            <fieldset block="MyAccountDetails" elem="AccountInfo">
+                <legend>Account Information</legend>
+                <div block="MyAccountDetails" elem="FieldWrapper">
+                    <div block="MyAccountDetails" elem="Field">
+                        { firstname }
+                        &nbsp;
+                        { lastname }
+                    </div>
+                    <div block="MyAccountDetails" elem="Field">{ email }</div>
+                    <div block="MyAccountDetails" elem="Field">
+                        Subscribed to newsletter:
+                        { is_subscribed ? ' Yes' : ' No' }
+                    </div>
+                    <div block="MyAccountDetails" elem="Actions">
+                        <button
+                          block="Button"
+                          mods={ { likeLink: true } }
+                          onClick={ () => this.changeState(STATE_EDIT_INFORMATION) }
+                        >
+                            Edit
+                        </button>
+                        <button
+                          block="Button"
+                          mods={ { likeLink: true } }
+                          onClick={ () => this.changeState(STATE_EDIT_PASSWORD) }
+                        >
+                            Change Password
+                        </button>
+                    </div>
+                </div>
+            </fieldset>
+        );
+    }
+
+    /**
+     * Render Customer Address block
+     * @param {String} addressType
+     */
+    renderAddress(addressType) {
+        const { customer: { addresses } } = this.props;
+
+        if (addresses) {
+            const correctAddress = addresses.filter(address => address[`default_${addressType}`])[0];
+
+            if (correctAddress) {
+                const {
+                    firstname,
+                    lastname,
+                    street,
+                    city,
+                    country_id,
+                    telephone
+                } = correctAddress;
+
+                return (
+                    <>
+                        <div block="MyAccountDetails" elem="Field">
+                            { firstname }
```

Comment:
Use string literals for this, please do not use special chars. `${firstname} ${lastname}`